tests/distributions/test_rejector.py

```diff
@@ -45,8 +45,8 @@ def compute_elbo_grad(model, guide, variables):
 @pytest.mark.parametrize('factor', [0.25, 0.5, 1.0])
 def test_rejector(rate, factor):
     num_samples = 100000
-    rates = torch.tensor(torch.tensor(rate).expand(num_samples, 1), requires_grad=True)
-    factors = torch.tensor(torch.tensor(factor).expand(num_samples, 1), requires_grad=True)
+    rates = torch.tensor(rate).expand(num_samples, 1).requires_grad_(True)
```

Comment:
We should never use `.expand(...).requires_grad_()`, since the resulting tensor will be a view. I believe these tests can be made sane by simply removing the `.requires_grad_()`.

Reply:
(If you actually do require grad, you should `.expand(...).contiguous().requires_grad_()`)